integration/src/main/java/ai/djl/integration/tests/ndarray/NDIndexTest.java

```diff
@@ -148,7 +148,9 @@ public void testSetNumber() {
     public void testSetScalar() {
         try (NDManager manager = NDManager.newBaseManager()) {
             NDArray original = manager.create(new float[] {1, 2, 3, 4}, new Shape(2, 2));
+            NDArray expected = manager.create(new float[] {0, 2, 3, 4}, new Shape(2, 2));
             original.setScalar(new NDIndex(0, 0), 0);
+            Assert.assertEquals(original, expected);
```

Comment:
nice catch, could we also add one more Assert after line154?

Reply:
Line 154 will throw an exception, because we try to setScalar to more than 1 elements. This behavior is indicated on line 147:

```java
@Test(expectedExceptions = IllegalArgumentException.class)
```

Reply:
otherwise, we can split these two tests(the normal case and the exception case) in different tests.